### PR TITLE
BXMSDOC-3389 Planning the Installation for DM and PAM 7.2

### PR DESCRIPTION
**Pull request by emmurphy1**
[RHPAM draft](http://file.ork.redhat.com/~emmurphy/rhpam-7.2-planning)
[RHDM draft](http://file.ork.redhat.com/~emmurphy/rhdm-7.2-planning)

### CODE REVIEW

File: assemblies/assembly_planning/main.adoc [hunk 1]
```diff
@@ -51,16 +51,22 @@ endif::[]
...
+=== {EAP} 7.2
+{EAP_LONG} ({EAP}) 7.2 is a certified implementation of the Java Enterprise Edition 7 (Java EE 7) full and web profile specifications. Red Hat JBoss EAP provides preconfigured options for features such as high availability, clustering, messaging, and distributed caching. It also enables users to write, deploy, and run applications using the various APIs and services that Red Hat JBoss EAP provides.
 
 You can install both {CENTRAL} and {KIE_SERVER} in a single {EAP} instance. However, in production environments you should install them in separate instances.
 
+=== Red Hat JBoss Web Server 5.0
+Red Hat JBoss Web Server is an enterprise-ready web server designed for medium and large applications, based on Tomcat. Red Hat JBoss Web Server provides organizations with a single deployment platform for Java Server Pages (JSP) and Java Servlet technologies, PHP, and CGI.
+
+You can install {KIE_SERVER} and the {HEADLESS_CONTROLLER} on Red Hat Web Server 5.0.
```
Comment (michelehaglund) [inline assemblies/assembly_planning/main.adoc]: @emmurphy1 - I'm sorry, but can you describe in great detail what the {HEADLESS_CONTROLLER} is? TEEEEEEEEEEE HEEEEEEEEEEEEEEE

**Review by domhanak — APPROVED** [review]
Looks good

**Review by domhanak — COMMENTED** [review]
@emmurphy1 I see `5.0` JWS version in some other place I see `5.0.1` Could you please give me an explanation why do we use 2 different versions across the documentation?

**Comment by emmurphy1** [comment]
@domhanak I removed most references to the JWS version to be consistent with EAP, and in the one remaining instance I specify 5.0.1 or higher. Please review and approve.

**Pull request closed without merging**